### DISCUSSION TITLE
Josh MVP Holtz & Felix Legend Krause ft. Community 🚀

### DISCUSSION THREAD

**Discussion by rogerluan** [fastlane/fastlane#16835] [General]
I'm so excited with this new Discussions feature in fastlane's GitHub page that I wanna kick off with a `💖 Thanks` post targeting our most active contributor [and now full-time Lead Maintainer](https://twitter.com/joshdholtz/status/1276535715019579399) @joshdholtz, and the one and only creator of _fastlane_, @KrauseFx 🚀🚀 

I started my iOS career nearly 7 years ago and have been a _fastlane_ user since pretty much the beginning (like ~5 years ago? 🤷‍♀️ somewhere around mid 2015). It's been a delight to use since then. @KrauseFx did this amazing Software Engineering project for college and little did he know it'd turn out to be a tool used by thousands of developers, including tech giants. So this thank-you note wouldn't be complete without a shout out to @KrauseFx! Thank you for all your constant efforts in the past few years, maintaining this amazing open source project that empowers engineers from all over the world to be more productive on a daily basis! 

@joshdholtz has been the most active member recently and was responsible for the App Store Connect API revamp that kicked off on June 16th 2020. He rushed to get a stable version of _fastlane_ out as soon as humanly possible and that's how we've been able to continue using _fastlane_. 

Of course those are not the only 2 folks that helped build this _thing_ 😉 
_fastlane_ main repository has 1044 contributors as of today, and each of those deserves a huge thank you ❤️ 

I'm very excited to see the 200+ new App Store Connect APIs that Apple is launching this year being implemented in _fastlane_ to make engineering teams even more productive! I hope to be able to contribute more to the project, even with the smallest things, as this has truly saved myself countless hours of work - nothing more fair than giving back to the community 🤟 

What a journey! 🚀

**Comment by joshdholtz** [comment]
@rogerluan Thank you so much for the love! 🙏  This was great message for myself, @KrauseFx, and the rest of the contributors to see 😊 We love _fastlane_ but we also love community that has grown around it. We keep doing what we do because of people like you ❤️

**Comment by joshdholtz** [comment]
I would also like to give a big, big shoutout to @snatchev for everything he does for _fastlane_ from the inside of Google ❤️ He is my hero 🏅
